dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -21,6 +21,7 @@ RUN apt-get update &&   \
         librrd-dev      \
         rrdtool         \
         python-smbus    \
+        python3-smbus   \
```

Comment:
Is this needed by the Nokia sonic-platform package? If so , I would prefer that you add it in the `install_requires` list of your setup.py file. This way it will get installed along with your package, and will not be installed if not needed.

Reply:
Yes, it is needed by the Nokia sonic-platform package and we see Arista using it also btw.    I attempted adding to install_requires list of setup.py file as you suggested, but it appears that mechanism is at runtime install time and did not work.  See error below:
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<pip._vendor.urllib3.connection.HTTPSConnection object at 0xb5731730>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution')': /simple/python3-smbus/
Build time install solution is preferred I think  -  putting the package in our debian another way - still working on alternatives other than the current Arista method multiple debians

Reply:
@carl-nokia: Ah yes. The vendor's sonic-platform package gets installed in the PMon container at runtime, thus you are correct, my suggestion will not work. I am OK with this solution for now. Sorry to trouble you.